packages/react-topology/src/behavior/withCreateConnector.tsx

```diff
@@ -87,14 +94,28 @@ const CreateConnectorWidget: React.FC<CreateConnectorWidgetProps> = observer(pro
       end: (dropResult: GraphElement, monitor: DragSourceMonitor, dragProps: any) => {
```

Comment:
Update `dragProps: CreateConnectorWidgetProps` to remove the use of `any`.
Also for the `drag` function above.